templates/company/test/company.js

```diff
@@ -243,6 +277,63 @@ contract('Company', ([_, owner, holder1, holder2]) => {
           })
         })
 
+        if (creationStyle === 'separate') {
+          context('when installing the payroll', () => {
+
+            context('when using the voting app for employee management', () => {
+              itHandlesInstanceCreationsProperly(true, true, false)
+
+              it('should have payroll app correctly setup', async () => {
+                assert.isTrue(await payroll.hasInitialized(), 'payroll not initialized')
+                assert.equal(await payroll.denominationToken(), PAYROLL_DENOMINATION_TOKEN)
+                assert.equal(await payroll.feed(), template.address)
```

Comment:
Perhaps we could use a constant for the feed address too, in the relevant contexts.

Reply:
Sure, but beware that it needs to be a contract since `isContract()` is run on that address.